WordPress/src/main/java/org/wordpress/android/ui/pages/PageParentFragment.kt

```diff
@@ -135,6 +183,8 @@ class PageParentFragment : Fragment() {
 
     override fun onSaveInstanceState(outState: Bundle) {
         linearLayoutManager?.let { outState.putParcelable(listStateKey, it.onSaveInstanceState()) }
+        previousSearchString = searchView.query.toString()
```

Comment:
I don't think the solution with a `previousSearchString` value in Fragment that is stored to saved instance state is correct (and clean) because right now you have a live data instance that holds the query and you also have it in the fragment. The advantage of ViewModel is that it survives the rotation. The only case when it's not enough is when the application is destroyed and recreated. In that case it's perfectly fine to lose the query (and search state). There are two options you can take to handle this correctly. The first one is simpler, you can expose `_searchQuery` and observe it. If the query is not empty and the search view is not showing, you'd show the search view and set the query. This would recreate the Search state on rotation. 
The other option (that's little cleaner) is to create a new class that would hold the UI state. This would be something like:
```
data class PageParentUiState(val pages: List<PageItem>, val searchQuery: String); 
```
You'd expose this instead of just `List<PageItem>` and do the logic in the `observe` method.